Scripts/script-CommonServerPython.yml

```diff
@@ -1203,6 +1203,46 @@ script: |-
       return True
 
 
+  def return_outputs(readable_output, outputs, raw_response=None):
+      """
+      This function replaces demisto.results(), makes the use more easy and intuitive.
```

Comment:
This function wraps the demisto.results(), makes the usage of returning results to the user more intuitively.